config/githooks/pre-push

```diff
@@ -35,8 +35,8 @@ cd $(git rev-parse --show-toplevel)
 # copyright check
 copyright=true
 if [[ "${option}" == "--copyright" || "${option}" == "" ]]; then
-  if git grep -l "^\(#\|//\).*\(\-2021\|\ 2010,\)" > matches.txt; then
-    echo "Please update the following files to Copyright (c) 2010-2022:"
+  if git grep -l "^\(#\|//\).*\(\-2022\|\ 2010,\)" > matches.txt; then
+    echo "Please update the following files to Copyright (c) 2010-2023:"
     cat matches.txt
     copyright=false
   fi
```

Comment:
This `git grep` command finds the Gecko copyright statements:
```console
git grep -n "^\(#\|//\).*\(\-2022\|\ 2010,\)"
general/gecko.cpp:22:// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC
general/gecko.cpp:52:// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and other
general/gecko.hpp:23:// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC
general/gecko.hpp:53:// Copyright (c) 2019-2022, Lawrence Livermore National Security, LLC and other
```
This produces the CI check failure. We can update the `grep` regex, e.g. with something like this:
```console
git grep -n "^\(#\|//\).*Copyright.*2010-20\(2[^3]\|[^2].\)"
```